minio/known_errors.py

```diff
@@ -0,0 +1,434 @@
+# Author: MrRedAmber (https://github.com/MrRedAmber)
+# Credits:
+# - http://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.htmlRESTErrorResponses
+# - https://github.com/riquito (pointing out my mistakes)
```

Comment:
While this is a nice gesture, it is not advisable to mention a github link of a github user. Github maintains the history of reviews if someone needs to look back.